DogeNet.DAL/DbInitializer.cs

```diff
@@ -0,0 +1,14 @@
+﻿// <copyright file="DbInitializer.cs" company="Leobit">
+// Copyright (c) Leobit. All rights reserved.
+// </copyright>
+
+namespace DogeNet.DAL
+{
+    public static class DbInitializer
+    {
+        public static void Initialize(DBContext context)
```

Comment:
Where do we use this class?

Reply:
To ensure that db is created
Should I delete it?

Reply:
I think you should delete it

Reply:
ok